test/distributed/test_c10d_gloo.py

```diff
@@ -2113,8 +2114,8 @@ def test_reducer_no_multi_replicas(self):
         num_replicas = 2
         models = [self._create_mixed_precision_model() for _ in range(num_replicas)]
         with self.assertRaisesRegex(
-            RuntimeError,
-            "Expected exactly one model replica.",
+            AssertionError,
```

Comment:
Wasn't sure if I should modify this test in some other way or just delete it

Reply:
I'd vote for removing this test, as `Reducer` ctor no longer takes list of lists.